ops/src/slack.rs

```diff
@@ -23,6 +24,9 @@ pub struct SlackAttachment {
 #[derive(Deserialize)]
 struct SlackResponse {
     ok: bool,
+    // ts is the message's ID that we can use to respond to messages
+    // https://api.slack.com/methods/chat.postMessage#examples
```

Comment:
ah, didn't realize this was a slack api response first time. 🤦 

looks like they discourage the use of this value though?

> Provide another message's ts value to make this message a reply. Avoid using a reply's ts value; use its parent instead.

though, not seeing `parent` in the examples. 🤔

Reply:
I believe the docs are saying to not use a _nested_ message from a slack thread for `ts` value when posting a new message.  They only want us to use the original/root of the thread message in the channel.  So we should be using it appropriately according to their docs.